internal/mtail/mtail.go

```diff
@@ -96,19 +98,23 @@ func (m *Server) initHTTPServer() error {
 	}
 
 	mux := http.NewServeMux()
-	mux.HandleFunc("/favicon.ico", FaviconHandler)
-	mux.Handle("/", m)
-	mux.Handle("/progz", http.HandlerFunc(m.r.ProgzHandler))
-	mux.HandleFunc("/json", http.HandlerFunc(m.e.HandleJSON))
+	if m.httpDebugEndpoints {
+		mux.HandleFunc("/favicon.ico", FaviconHandler)
+		mux.Handle("/", m)
```

Comment:
These two previous are Info endpoints

Reply:
Moved this to the info endpoint section. Question would be, Should we render a different template in case debugging is disabled (internal/mtail/httpstatus.go) as the debug links would be dead then ?

Reply:
I think we can pass the boolean into the template itself and only render those links if the boolean is set, right?

https://github.com/google/mtail/blob/main/internal/mtail/httpstatus.go#L36 copy `m.httpDebugEndpoints` into this struct and then update the `statusTemplate` to use `{{ if .httpDebugEndpoints }}` per https://pkg.go.dev/text/template